server/routes/overview/overviewView.ts

```diff
@@ -1,21 +1,24 @@
-import type { PrisonerSummary } from 'viewModels'
+import type { PrisonerSummary, SupportNeeds } from 'viewModels'
```

Comment:
Re: my comment above - revert the changes in this class, but make the same changes to `SupportNeedsView` instead 👍